src/main/webapp/gameFunctions.js

```diff
@@ -0,0 +1,7 @@
+function getUserQuizStatus() {
+    fetch("/user-quiz-status-servlet").then(response => response.json()).then((tasks) => {
+        if(tasks==false){
```

Comment:
Nit: Instead of comparing `x == false`, do `!x`. This is mostly for the sake of style.